src/www/components/popup/PopupComp.jsx

```diff
@@ -0,0 +1,183 @@
+import "./PopupComp.scss";
+import React, { Component } from "react";
+
+import Input from "../../components/input/Input";
+import axios from "axios";
+import Popup from "reactjs-popup";
+import ImageUploader from "react-images-upload";
+import ScrollArea from "react-scrollbar";
+import { Redirect } from "react-router-dom";
+import { withTranslation } from "react-i18next";
+
+class PopupComp extends Component {
+  state = {
+    teamName: "",
+    description: "",
+    teamCategories: [],
+    teamPicture: "",
```

Comment:
`teamPicture` is probably not a string, so we should better initialize this as undefined